### DISCUSSION TITLE
Compilation error

### DISCUSSION THREAD

**Discussion by sanjali-agrawal** [opengisch/QField#3252] [Q&A]
On following the guidelines specified in development documentation https://github.com/opengisch/QField/blob/master/doc/dev.md, I was trying to compile the QField android app, but I am getting stuck and getting some errors
<img width="1066" alt="Screenshot 2022-08-26 at 10 45 08 AM" src="https://user-images.githubusercontent.com/61723791/186827287-b5a80a8c-a9a7-4ce9-b82b-068a3b1c18cc.png">

I tried fixing them by making some changes in the build-vcpkg.sh file, but then new errors came up
<img width="991" alt="Screenshot 2022-08-26 at 10 46 50 AM" src="https://user-images.githubusercontent.com/61723791/186827460-d47eef86-ea6a-4bf5-b0e8-6f2678dcf887.png">

Can somebody help me figure out what I am doing wrong in this compilation?

**Comment by m-kuhn** [comment]
Can you check with the latest version of the dev document and provide the complete command that you were running?
